File: jobs/2026-05-26__11-16-44/dynamic_fanout_py__KSPiKvZ/artifacts/user/myproject/worker.py
from workflows import hatchet, process_batch_workflow, square_item_workflow


def main() -> None:
    worker = hatchet.worker(
        "dynamic-fanout-worker",
        workflows=[square_item_workflow, process_batch_workflow],
    )
    worker.start()


if __name__ == "__main__":
    main()

File: jobs/2026-05-26__00-25-55/linear_dag_ts__mpdpwGY/artifacts/user/myproject/src/worker.ts
import { Hatchet } from '@hatchet-dev/typescript-sdk';
import { workflow } from './workflow';

const hatchet = Hatchet.init();
const runId = process.env.ZEALT_RUN_ID || 'local';

async function main() {
  const worker = await hatchet.worker(`linear-dag-worker-${runId}`, {
    workflows: [workflow],
  });
  await worker.start();
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
